app/decorators/pseud_decorator.rb

```diff
@@ -11,6 +19,7 @@ def self.decorate_from_search(results, search_hits)
     end
   end
 
+  # TODO Either eliminate this function or make it work properly.
```

Comment:
Just wondering about this comment! Did something new get broken in pseud search?

Reply:
Nothing new got broken. I think the function just never got finished, and since it never gets called, no one noticed that it always errors. (`NameError: undefined local variable or method 'work_counts' for PseudDecorator:Class`)

I added more details about how to fix it in the comment -- I know TODOs have a tendency to get lost in the shuffle if there's no associated bug report, but I wanted to mark that function so that anyone who tries to use it in the future knows that it's not done.